lib/moon_web/router.ex

```diff
@@ -78,12 +78,12 @@ defmodule MoonWeb.Router do
         live("/components/date/single-date", MoonWeb.Pages.Components.Date.SingleDatePage)
         live("/components/date/range-date", MoonWeb.Pages.Components.Date.RangeDatePage)
 
-        live("/components/dialog/modal", MoonWeb.Pages.Components.Dialog.ModalPage)
+        live("/components/modal", MoonWeb.Pages.Components.ModalPage)
```

Comment:
Should you update component itself?

Reply:
I didnt't change componets, changes only on moon_web directory.

Reply:
Ok, but there is 1:1 mapping between documentation page and real component. You just can not change 1 without changing other.

Reply:
but moving components themselves is a breaking change. any reason for this?

Reply:
Let's break it then. It is search and replace fix. Let's try to keep things nice and tidy.